wiki/Submission/id.md

```diff
@@ -0,0 +1,27 @@
+# Submission
+
+[Beatmaps](/wiki/Beatmaps) dapat diunggah ke situs osu! melalui [in-game editor](/wiki/Beatmap_Editor). Mengunggah sebuah beatmap memberikan sebuah perhatian kepada pengguna lain dan berpotensi dapat mengikuti kategori [Ranked](/wiki/Beatmaps#ranked) atau [Loved](/wiki/Beatmaps#loved).
+
+![](img/bss_warning.png "Tampilan Beatmap Submission Sistem")
```

Comment:
``Beatmap Submission Sistem`` --> ``Beatmap Submission System``

should apply to other concurrent instances of this as well i guess